README.rst

```diff
@@ -536,7 +536,7 @@ you're testing against multiple versions of Python.  Next, run tox:
 
   tox
 
-Acknowledgements
+Acknowledgments
 ----------------
```

Comment:
Please also correct the underline for the heading to match the new heading length.
```suggestion
---------------
```